src/System.CommandLine.Subsystems.Tests/ValueSubsystemTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using System.CommandLine.Directives;
+using System.CommandLine.Parsing;
+using Xunit;
+
+namespace System.CommandLine.Subsystems.Tests;
+
+public class ValueSubsystemTests
+{
+    [Fact]
+    public void Value_is_always_activated()
+    {
+        CliRootCommand rootCommand = [
+            new CliCommand("x")
+            {
+                new CliOption<string>("--opt1")
+            }];
+        var configuration = new CliConfiguration(rootCommand);
+        var subsystem = new ValueSubsystem();
+        var input = "x --opt1 Kirk";
+        var args = CliParser.SplitCommandLine(input).ToList();
+
+        Subsystem.Initialize(subsystem, configuration, args);
+        var parseResult = CliParser.Parse(rootCommand, args[0], configuration);
+        var isActive = Subsystem.GetIsActivated(subsystem, parseResult);
+
+        isActive.Should().BeTrue();
+    }
+
+    [Fact]
+    public void ValueSubsystem_returns_values_that_are_entered()
+    {
+        var consoleHack = new ConsoleHack().RedirectToBuffer(true);
+        CliOption<int> option1 = new CliOption<int>("--intValue");
+        CliRootCommand rootCommand = [
+            new CliCommand("x")
+            {
+                option1
+            }];
+        var configuration = new CliConfiguration(rootCommand);
+        var pipeline = Pipeline.CreateEmpty();
+        pipeline.Value = new ValueSubsystem();
+        const int expected1 = 42;
```

Comment:
```suggestion
        const int expected = 42;
```
There seems to be only one `expected` variable, can this drop the suffix? (Great choice of numbers btw)

This exists multiple places in this file

Reply:
Thanks!